src/AppBundle/Resources/config/routing/api/routing_api_oauth2.yml

```diff
@@ -18,8 +18,6 @@ api_oauth2_new_deck:
     methods: [POST]
     defaults:
         _controller: AppBundle:Oauth2:newDeck
-    requirements:

```

Comment:
Realized this was causing the 'id' to show up as a requirement in the docs, so went ahead and took it out.